src/JuliusSweetland.OptiKey/UI/Controls/CK20Page.xaml.cs

```diff
@@ -1130,7 +622,7 @@ public string CKText_10
         public string CKBaCo_10
         {
             get { return (string)GetValue(CKBaCo_10Property); }
-            set { SetValue(CKBaCo_10Property, dec2hex(value)); }
+            set { SetValue(CKBaCo_10Property, value); }
         }
 
         public static readonly DependencyProperty CKImSo_10Property =
```

Comment:
For each of the CKImSo_xy setters (a few lines below, I can't comment there) could you please move the logic into one place outside of the setter, just like you did for rgb2hex. This makes it easier to modify if the logic changes later.

Reply:
Thanks for reminding me about this. The logic is now done on lines [168-170](https://github.com/zelmon64/OptiKey/blob/29b1d2ace719f6e1f8a2f0b50537a164227aab2b/src/JuliusSweetland.OptiKey/UI/Controls/CK20Page.xaml.cs#L168-L170).